tests/smoke/base.sh

```diff
@@ -40,7 +40,7 @@ export PATH=$(echo $PATH| sed -E "s/[^:]+\/micromamba\/[^:]+//g")
 export PATH=$(echo $PATH| sed -E "s/[^:]+\/anaconda3\/[^:]+//g")
 export PATH="${CONDA_PREFIX}:${CONDA_PREFIX}/bin:$PATH"
 echo "[II] included env conda to the PATH"
-
+COMMAND_PREFIX= 
```

Comment:
Added here by mistake?

Reply:
actually it is not necessary, because bash will consider any non declared variable as empty .. but I think it is good (as a "code documentation") to have it declared and assigned to empty at the beginning ... so if anyone wants something similar they will use the same variable

Reply:
Thanks for the clarification! Did not know that!